README.md

```diff
@@ -37,10 +37,12 @@ python index.py
 
 ## Development
 
-If your development environment doesn't already automatically use PEP8 for formatting, you can 
-install a git hook to automatically format Python files as part of the commit process. To do so,
-run this once:
+Consistent PEP8 compliant formatting can be ensured by enabling pre-commit
+git hook.
 
     ```
-    git config core.hooksPath .githooks
```

Comment:
The `.githooks` directory or `.githooks/pre-commit` file should probably also be deleted, so that both black and pep8 don't run back-to-back.

Reply:
Good point. I have done so.